tests_e2e/test_create.py

```diff
@@ -0,0 +1,97 @@
+import os, pytest, requests, time
+from threading import Thread
+from selenium import webdriver
+from selenium.webdriver.common.action_chains import ActionChains
+from dotenv import find_dotenv, load_dotenv
+import todo_app.app as app
+
+#from requests.models import Response
+# Loading environment variables 
+file_path = find_dotenv('.env')
+load_dotenv(file_path, override=True)
+
+BASE_URL = "https://api.trello.com/1/"
+API_KEY = os.environ.get('TRELLO_API_KEY')
+TOKEN = os.environ.get('TRELLO_API_SECRET')
+
+def create_trello_board():
+    params = {
+        'key': API_KEY,
+        'token': TOKEN,
+        'name': "E2E Testing",
+    }
+    response = requests.post(BASE_URL + 'boards', params=params)
+    return response
+
+def delete_trello_board(board_id):
+    params = {
+        'key': API_KEY,
+        'token': TOKEN,
+    }
+    response = requests.delete(BASE_URL + f'boards/{board_id}', params=params)
+    return response
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Create the new board & update the board id environment variable
+    create_board_response = create_trello_board()
+    if create_board_response.status_code == 200:
+        board_id = create_board_response.json()['id']
+        board_name = 'E2E Testing'
+        os.environ['TRELLO_BOARD_NAME'] = board_name 
+    else:
+        print(f'create board failure {create_board_response.status_code}')
```

Comment:
Maybe raise an exception instead? You don't want to carry on with the test in this case.

```suggestion
        raise Exception(f'create board failure {create_board_response.status_code}')
```

Reply:
thank you